contracts/PodSponsorshipFactory.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.6.4;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/Initializable.sol";
+import "@pooltogether/pooltogether-contracts/contracts/external/openzeppelin/ProxyFactory.sol";
+import "./PodSponsorship.sol";
+
+contract PodSponsorshipFactory is Initializable, ProxyFactory {
+
+    PodSponsorship internal sponsorshipTemplate;
+
+    event PodSponsorshipCreated(address indexed token, address indexed pod);
```

Comment:
This event is already provided by the ProxyFactory: `ProxyCreated`.  It has the address, and we can dig out the pod address from the created contract.